generate_test_images.py

```
@@ -2,86 +2,93 @@
+    elif name == "text":
         img = np.ones((height, width, 3), dtype=np.uint8) * 255
         pil_img = Image.fromarray(img)
         draw = ImageDraw.Draw(pil_img)
-        
+
         try:
-            font = ImageFont.truetype("/usr/share/fonts/truetype/dejavu/DejaVuSans-Bold.ttf", 20)
+            font = ImageFont.truetype(
+                "/usr/share/fonts/truetype/dejavu/DejaVuSans-Bold.ttf", 20
+            )
         except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)